ivy_tests/test_ivy/test_functional/test_core/test_manipulation.py

```diff
@@ -340,7 +335,11 @@ def test_roll(
 ):
 
     value_dtype, value = dtype_value
-    shift = shift[1] if isinstance(shift[1], int) else tuple(shift[1])
+    if isinstance(shift[1], int):
```

Comment:
could you add a comment explaining the purpose of this `if` `else` logic?

Reply:
Added comment explaining logic